cmd/validate/image.go

```diff
@@ -249,7 +249,15 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 							if len(parts) < 2 {
 								log.Errorf("Incorrect syntax for --extra-rule-data")
 							}
-							unmarshaled[parts[0]] = parts[1]
+							if source_utils.SourceIsFile(parts[1]) && utils.HasJsonOrYamlExt(parts[1]) {
```

Comment:
It might make sense use `--extra-rule-data=file:...` syntax, so we can pull in from other kinds of services, not just from file system, in that case relying on `github.com/enterprise-contract/ec-cli/internal/policy/source.PolicySource` might be a good solution here